### PR TITLE
Login Done

### PR DESCRIPTION
**Pull request by PK4ever**

### CODE REVIEW

File: classes/artifacts/laker-polling/exploded/laker-polling-0.1.war/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Manifest-Version: 1.0
```
Comment (mwiater1) [inline classes/artifacts/laker-polling/exploded/laker-polling-0.1.war/META-INF/MANIFEST.MF]: What does this file do? It seems to be the manifest for the war file but it definitely should not be where it currently is placed. This file is supposed to be generated along with and packaged into the WAR.

File: grails-app/assets/clientapp/css/libs/malin/animated-headlines.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+/*
```
Comment (mwiater1) [inline grails-app/assets/clientapp/css/libs/malin/animated-headlines.css]: This entire file looks like a user specific error message. What is it?

File: grails-app/assets/clientapp/js/chat.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
+$(document).ready(function(){
```
Comment (mwiater1) [inline grails-app/assets/clientapp/js/chat.js]: This js file does not seem relevant to the project. What is it for?

File: grails-app/assets/clientapp/js/google-tracking.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```
Comment (mwiater1) [inline grails-app/assets/clientapp/js/google-tracking.js]: I wasn't aware we are including google analytics on the app. Who has access to the data?

File: grails-app/assets/clientapp/js/mediumClient.js [hunk 5]
```diff
@@ -0,0 +1,294 @@
+var medium = require("../")
```
Comment (mwiater1) [inline grails-app/assets/clientapp/js/mediumClient.js]: This js file does not seem relevant to the project. What is it for?

File: grails-app/views/application/index.gsp [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <!--for SEO: tells search engines that we are using ajax calls and so it should crawl our webapp appropriately.-->
+    <meta name="fragment" content="!">
+    <title ng-bind="title">Laker Polling</title>
+    <meta property="description"
+          content="Clicker System for SUNY Oswego built by Lakers.">
+    <base href="/">
+    <meta name="keywords"
+          content="Clicker System for SUNY Oswego built by Lakers"/>
+    <meta property="og:url" content="laker-polling.com">
```
Comment (mwiater1) [inline grails-app/views/application/index.gsp]: We do not own the domain "laker-polling.com"

File: grails-app/views/application/index.gsp [hunk 7]
```diff
@@ -0,0 +1,261 @@
...
+    </div>
+    <!--END FOOTER-->
+
+    <!-- =================== START BELOW FOLD SCRIPTS =================== -->
+    <!-- =================== START BELOW FOLD SCRIPTS =================== -->
+
+    <!-- ================= START ANGULAR APP DEPENDENDCIES ================== -->
+    <!-- ================= START ANGULAR APP DEPENDENDCIES ================== -->
+
+    %{--<script src="/grails-app/assets/clientapp/js/libs/angular-infinite-scroll/angular-infinite-scroll.js"></script>--}%
```
Comment (mwiater1) [inline grails-app/views/application/index.gsp]: Looks like this line is commented out and implemented below. Please remove.

File: grails-app/assets/clientapp/js/app/config.js [hunk 8]
```diff
@@ -0,0 +1,107 @@
+angularApp
+    .config(['$routeProvider', '$locationProvider', function AppConfig($routeProvider, $locationProvider) {
+
+        var auth2; // this variable needs to be persistent across pages or the code below it needs to run on all pages
+        //otherwise no action on google auth can be taken.
+
+        gapi.load('auth2', function () {
+           auth2 = gapi.auth2.init({
+               client_id: '896100416043-v0cvdf52tteag7ha8939fog24sr7bm2g.apps.googleusercontent.com',
```
Comment (sschneller) [inline grails-app/assets/clientapp/js/app/config.js]: Is this an api key or something?  This shouldn't be version controlled if so, for safety issues

File: grails-app/assets/clientapp/img/close-x.svg [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Capa_1" x="0px" y="0px" width="512px" height="512px" viewBox="0 0 612 612" style="enable-background:new 0 0 612 612;" xml:space="preserve">
+<g>
+	<g id="cross">
+		<g>
+			<polygon points="612,36.004 576.521,0.603 306,270.608 35.478,0.603 0,36.004 270.522,306.011 0,575.997 35.478,611.397      306,341.411 576.521,611.397 612,575.997 341.459,306.011    " fill="rgba(0,0,0,0.44)"/>
+		</g>
+	</g>
+</g>
+<g>
```
Comment (sschneller) [inline grails-app/assets/clientapp/img/close-x.svg]: What's up with all these open/close g tags?

File: grails-app/assets/clientapp/js/app/config.js (hunk 8 above)
Comment (jregistr) [inline grails-app/assets/clientapp/js/app/config.js]: This is an api key, however, it's public and can be committed.

File: grails-app/assets/clientapp/css/libs/font-awesome-4.3.0/less/animated.less [hunk 10]
```diff
@@ -0,0 +1,34 @@
+// Animated Icons
```
Comment (mwiater1) [inline grails-app/assets/clientapp/css/libs/font-awesome-4.3.0/less/animated.less]: This file, along with the entire "font-awesome-4.3.0" directory looks like it is not being used. You already have the "font-awesome.min.css" file in a different folder that is being used. I think this entire folder should be removed unless there is a reason to keep it and the files it has.

File: grails-app/controllers/edu/oswego/cs/lakerpolling/controllers/ApplicationController.groovy [hunk 11]
```diff
@@ -1,16 +1,79 @@
 package edu.oswego.cs.lakerpolling.controllers
```
Comment (mwiater1) [inline grails-app/controllers/edu/oswego/cs/lakerpolling/controllers/ApplicationController.groovy]: .groovy files should not be modified by you guys. There are not any changes in the file itself so im not sure why git is showing this as a change. The best option to do this is to checkout these files from origin. Therefore git will recognize that these files have not changed at all. I'm only putting this comment on this groovy file. Please do this to all .groovy files git shows as modified.

**Comment by mwiater1** [comment]
Closed.
